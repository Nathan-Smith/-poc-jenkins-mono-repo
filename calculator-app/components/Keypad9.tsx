import { KeypadButton } from './KeypadButton'

export function Keypad9(): JSX.Element {
  return (
    <KeypadButton testid="keypad-9" gridArea="keypad-9">
      9
    </KeypadButton>
  )
}
