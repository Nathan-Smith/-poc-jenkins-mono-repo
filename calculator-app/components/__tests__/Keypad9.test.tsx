import React from 'react'
import renderer, { act, ReactTestRenderer } from 'react-test-renderer'

import { Keypad9 } from '../Keypad9'

describe('Keypad9', () => {
  test('renders', async () => {
    let component: ReactTestRenderer | undefined
    act(() => {
      component = renderer.create(<Keypad9/>)
    })

    expect(component?.toJSON()).toMatchSnapshot()
  })
})
