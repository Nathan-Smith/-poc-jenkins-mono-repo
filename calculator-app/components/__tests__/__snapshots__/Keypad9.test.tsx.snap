// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Keypad9 renders 1`] = `
<button
  className="mdc-button mdc-button--outlined keypadButton"
  data-testid="keypad-9"
  onClick={[Function]}
  style={
    Object {
      "gridArea": "keypad-9",
    }
  }
>
  <span
    className="mdc-button__ripple"
  />
  <span
    className="mdc-button__label"
  >
    9
  </span>
</button>
`;
